Find a functionally equivalent code:android.graphics.Canvas.isOpaque:COMMENT
<android.graphics.Canvas: boolean isOpaque()>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Return true if the device that the current layer draws into is opaque
 * (i.e. does not support per-pixel alpha).
 *
 * @return true if the device that the current layer draws into is opaque
 */

@@ -1 +1,3 @@
-
+{
+    return native_isOpaque(mNativeCanvasWrapper);
+}

